.devcontainer/create-tarball/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
```

Comment:
Can the workdir of the container be set to the /workspace folder instead of the installer repo?  It would be useful to see the tarball in the Explorer for both containers.